docs/02-iaas/mandatory-and-supported-os-services.md

```diff
@@ -0,0 +1,55 @@
+# SCS mandatory and supported OpenStack services
+
+There are many OpenStack APIs and their services that can be applied on IaaS-Level.
+These services have differences in the quality of their implementation and liveness and some of them may be easily omitted when creating an IaaS-Deployment.
+To fulfill all SCS-provided standards there are only some of these APIs required.
+More but not all OpenStack services are tested or integrated in the reference implementation.
+This document will give readers insight about how the SCS looks at all the OpenStack services.
+If a cloud provides all mandatory and maybe some supported OpenStack APIs and implementation of their services it can be tested for SCS-compliance.
+Any unsupported services will not be tested.
+
+## Mandatory OpenStack services
+
+The following OpenStack services MUST be present in SCS-compliant IaaS-Deployments:
+
+| OpenStack Service | description |
+|-----|-----|
+| **Cinder** | Block Storage service |
+| **Glance** | Image service |
+| **Keystone** | Identity service |
+| **Neutron** | Networking service |
+| **Nova** | Compute service |
+| **Octavia** | Load-balancer service |
+| **Placement** | Hardware Describing Service for Nova |
+| **S3 API object storage** | No formal standard exists, many implementations: Swift, RadosGW, minio... |
```

Comment:
note: the reader might wonder why this is included when there is no real standard for S3 APIs and the existing implementations all slightly differ.

I think it's safe to say that it is still included because of it's general usefulness and for many usecases the implementations are similar enough to maybe not matter.

It might be worth it though, to add some kind of warning for powerusers, that they should research if their S3 feature is supported by the used implementation.

That being said: Would it make sense to mandate, that the used S3 Implementation must be at least publicly documented so users _can_ actually  research if the implementation is fit for their purpose?

I think that could be important, at least for power users and should not put too much of a burden on the CSP. Maybe this can also already be discovered via existing APIs within S3 itself? But I really don't know about the latter part.